### DISCUSSION TITLE
How to compare two fuel in one script?

### DISCUSSION THREAD

**Discussion by Hannanaaaaa** [firemodels/fds#13307] [Q&A]
Hey, I’m new to FDS and still learning how to use it. can you advise me how to compare two fuels in one script? i want compare n-heptane and n-octana. thank you

**Comment by mcgratta** [comment]
Did you try looking at the User's Guide? Is there something not clear?

**Reply by Hannanaaaaa** [reply]
Sorry, I've tried to look for it in the user's guide but I'm still confused about which part explains that part, if you would like, could you tell me what part it is?

**Comment by mcgratta** [comment]
![image](https://github.com/user-attachments/assets/a5467592-4b14-433c-9069-44a4fe2ec4c9)
